### PR TITLE
updated mapping to use new street combined property

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

This PR updates the JSON ERB mapping for IVC form 10-7959f-1 so the new `streetCombined` property is used rather than `street` when stamping PDFs. This new property includes streets 1, 2, and 3 concatenated together. 

Previously we were not stamping streets 2 and 3 onto the PDF, which could result in leaving information out if the user filled in those fields on the frontend.

## Related issue(s)

- [Frontend change](https://github.com/department-of-veterans-affairs/vets-website/pull/31590)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/90729

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots

| |Stamped|
|-|-|
|before|Screenshot 2024-08-26 at 13 36 59|
|after|Screenshot 2024-08-26 at 13 37 07|

## What areas of the site does it impact?
This form only

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

NA

### CODE REVIEW

**Review by balexandr — DISMISSED** [review]

**Review by ericboehs — APPROVED** [review]

**Review by cloudmagic80 — DISMISSED** [review]

**Comment by michaelclement** [comment]
Waiting to merge this until after 10-7959f-1 staging review (scheduled for 8/30/2024)

**Comment by michaelclement** [comment]
@ericboehs Could I have a re-approval when you get a chance? I had to update a unit test. Thank you!

**Review by ericboehs — APPROVED** [review]

**Pull request merged**
